Analyse_Birds.qmd

```diff
@@ -436,11 +450,389 @@ for (i in 1:NSim) {
+data per modality. It will therefore be necessary to remove these data before continuing
+the analysis
+
+Factor Year: The experimental design is not balanced (from 24 to 448 data), but
+the analysis can be performed because there are enough data in each modality (> 10).
+
+
+### Removal of poorly represented modalities (< 10 data)
+
+```{r Ini_delete_modalites}
+#| echo: true
+#| code-fold: true
+#| code-summary: "Show the code"
+
+# Count the number of occurrences for each modality in data$Species
+species_count <- table(data$Species)
+
+# Keep only terms with 10 or more occurrences
+data <- data %>% filter(Species %in% names(species_count[species_count >= 10]))
+
+# Show filtered data
+head(data)
+summary(data$Species)
+```
+  
+  
+### Analyzing the potential relationship between Y vs Xs
+
+We can graphically analyze the possible relationships between Y and X. This 
+graphical analysis of the relationships between Y and X **does not allow us to predict the significance of the relationship**. Statistical modeling remains the only available means to identify whether a relationship exists or not.
+
+
+```{r Ini_datagraph, fig.height=3, fig.width=6}
+#| echo: true
+#| code-fold: true
+#| code-summary: "Show the code"
+
+# Boxplot 
+par(mfrow=c(1,2))
+boxplot(data$Initiation_Date~data$State, varwidth = TRUE, ylab = "Initiation date", xlab = "State", col='darkgrey', main = "")
+boxplot(data$Initiation_Date~data$Habitat, varwidth = TRUE, ylab = "Initiation date", xlab = "Habitat", col='darkgrey', main = "")
+boxplot(data$Initiation_Date~data$Species, varwidth = TRUE, ylab = "Initiation date", xlab = "Species", col='darkgrey', main = "")
+boxplot(data$Initiation_Date~data$Year, varwidth = TRUE, ylab = "Initiation date", xlab = "Year", col='darkgrey', main = "")
+```
+
+From the first visualizations, we could make the hypothesis that Year and Species
+influence Initiation_Date.
+
+
+### Analysis of possible interactions between the four Xs
+
+The interaction between two factors can only be tested if the factors are
+crossed (all the modalities of one factor are represented in all
+the modalities of the other factor and vice versa = a complete factorial design).
+To estimate the presence of interactive effects, we will develop a graphical approach.
+
+
+```{r Ini_dataInter, fig.height=4, fig.width=7}
+#| echo: true
+#| code-fold: true
+#| code-summary: "Show the code"
+
+# Interaction table
+table(data$State,data$Habitat)
+table(data$State,data$Species)
+table(data$State,data$Year)
+table(data$Habitat,data$Species)
+table(data$Habitat,data$Year)
+table(data$Species,data$Year)
+
+```
+
+
+Subsequently, we will not analyze the interactions between the factors since 
+
+### Search for candidate model
+
+```{r Ini_fullmodel}
+#| echo: true
+#| code-fold: true
+#| code-summary: "Show the code"
+
+# Model formulation
+mod1<-lm(Initiation_Date~State+Habitat+Species+Year,data=data)
```

Comment:
Do we have to precise the hypothesis for using lm?

Reply:
There is a model validation section below which includes the conditions.